### PR TITLE
Bug 1950409: Update v1beta1 references to v1

### PR DESCRIPTION
**Pull request by damemi**

### CODE REVIEW

**Review by damemi — COMMENTED** [review]
/cc @ingvagabund

**Comment by ingvagabund** [comment]
Good catch!!!
/lgtm

File: pkg/operator/target_config_reconciler.go [hunk 1]
```diff
@@ -146,7 +146,7 @@ func (c *TargetConfigReconciler) manageRole(descheduler *deschedulerv1.KubeDesch
 	required.Namespace = descheduler.Namespace
 	required.OwnerReferences = []metav1.OwnerReference{
 		{
-			APIVersion: "v1beta1",
```
Comment (ingvagabund) [inline pkg/operator/target_config_reconciler.go:149]: we might actually define a function to generated the owner here:
```
function ownerRef(descheduler *deschedulerv1.KubeDescheduler) metav1.OwnerReference {
	return {
		APIVersion: "v1",
		Kind:       "KubeDescheduler",
		Name:       descheduler.Name,
		UID:        descheduler.UID,
	}
}
```

**Pull request merged**